### DISCUSSION TITLE
Mapping mouse coordinates only when it enters the rectangle

### DISCUSSION THREAD

**Discussion by monolithMktg** [cables-gl/cables_docs#352] [Answers & Questions] [answered]
I want to map my mouse coordinates from 0-1 only relative to the x and y axis of the rectangle, not the entire canvas. I am not able to achieve that.  The bulge coordinates are getting offset when the cursor leaves the rectangle. Could you assist?

**Comment by steam0r** [comment]
hey, are you maybe looking for MapRange?

**Reply by monolithMktg** [reply]
I am trying to trigger a MapRange 'after' the mouse has entered a small rectangle at the middle of the canvas. Currently, the mouse X and Y coords are registered the moment it enters the canvas.

**Comment by pandrr** [comment, marked as answer]
you can use interactive rectangle for this

PLEASE DONT INVITE ME INTO PATCHES WITHOUT TALKING TO ME BEFORE

**Reply by monolithMktg** [reply]
I'm sorry for that. Thanks.
